doc/terminology.rst

```diff
@@ -0,0 +1,51 @@
+.. _terminology:
+
+.. https://github.com/pydata/xarray/issues/2410
+.. https://github.com/pydata/xarray/issues/1295
+
+Terminology
+===========
+
+*Xarray terminology differs slightly from CF and mathematical conventions, and therefore using xarray, understanding the documentation, and parsing error messages is easier once key terminology is defined. This glossary was designed so that more fundamental concepts come first. Thus for new users, this page is best read top-to-bottom. Throughout the glossary,* ``arr`` *will refer to an xarray* :py:class:`DataArray` *in any small examples. For more complete examples, please consult the relevant documentation.*
+
+----
```

Comment:
can you add a screenshot of how it looks?

Reply:
Yeah, will do once I've gone through the current feedback, integrated it, and rebuilt.